README.md

```diff
@@ -579,6 +582,39 @@ async function customMiddleware(ctx, next) {
 }
 ```
 
+## Error Handling
+
+While interacting with the Okta API, the client may experience the following HTTP errors:
+
+* 429 - This is a rate limiting exception, please see [Rate Limiting at Okta] for complete details on our rate limiting strategy.
+* 503 - Service unavailable.  An unlikely situation, see [Okta Trust] for real-time service updates.
+
+In either case you will want to retry the request.  There are two ways you can handle retries, described below.
+
+### Custom Fetch Implementation
+
+If you are running into many rate limiting errors across several APIs we suggest you implement your own [Fetch interface].  The SDK will delegate all HTTP calls to your fetch interface and you must return a promise that resolves to a [Fetch response].  This allows you to write a backoff strategy for your specific needs.
+
+To provide your own fetch implementation, provide it as the `fetch` option:
+
+```javascript
+const client = new okta.Client({
+  orgUrl: 'https://dev-1234.oktapreview.com/'
+  token: 'xYzabc', // Obtained from Developer Dashboard
+  fetch: (uri, init) => {
+    // Perform the the http request defined by input, init
+    // On error, decide how to backoff and retry
+    // You MUST return a promise that resolves to a Fetch response specification
+  }
+});
+```
+
+We suggest evaluating libraries like [node-backoff] and [node-retry] to help with building your strategy.
+
+### Method-level Handling
+
+If you are using the SDK for only a few methods, you may want to simply handle the error in the callback handler and do a simple retry.  All methods that resolve with an error will have the necessary HTTP header information to make a decision about how to retry.  Please see the [Advanced Subscription](#advanced-subscription) example above for header parsing details.
```

Comment:
Is there an example of this with method-level error handling? Advanced subscription shows me how to parse it, but I don't see other examples of getting a response error in the method examples. I'm assuming I can catch any of the methods and it'll have 429 and the res details, but I don't see an example of that.

Reply:
That's correct, I can make this more clear.